### PR TITLE
Us3 homepage

### PR DESCRIPTION
**Pull request by MorganeL-pro**
added html and css for the homepage.
Added components header and footer

### CODE REVIEW

**Comment by elodiedaubie** [comment]
Hello,
I have 2 errors on the W3C validator :

**Comment by elodiedaubie** [comment]
Hello,
The responsiveness and code quality is ok for me. 
Nevertheless, I have 2 errors on the HTML validator.
1 - There is two h1 on the page, we should consider using 3 levels of titles (from H1 to H3).
2 - the second one is about the script written in the header. It looks like to be Symfony's way, so I'm not sure we can fix this.
Also, we might change the image because it's pixelated, I'm not sure it's a good idea to show the page like this to the customer.

**Comment by MorganeL-pro** [comment]
For the script error, it seems it comes from Bootstrap and i can't do nothing.
For the h1 tag, we have too on the homepage but it's not a pb according to this website https://referenceur.be/multiples-balises-h1-bonne-mauvaise-idee.html.

**Review by elodiedaubie — APPROVED** [review]
Ok for me

File: assets/styles/variables.scss [hunk 1]
```diff
@@ -1,2 +1,9 @@
-$wild-color: #ff6c71;
-$light: #fffbfb;
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300&display=swap');
```
Comment (RomainClair) [inline assets/styles/variables.scss:1]: You should probably host your assets

**Review by RomainClair — APPROVED** [review]

**Pull request merged**